smaug/python/ops/ops_test.py

```diff
@@ -4,7 +4,7 @@
 
 import unittest
 
-from smaug.python import smaug_test
+from smaug.python.smaug_test import *
```

Comment:
same here.

Reply:
Removed the import * in the latest commit.